### ISSUE TITLE
[Bug]: System accent theme doesn't match chrome colors

### ISSUE THREAD

**Issue by cantunborn** [edelvarden/material-fox-updated#81]
### What happened?

After changing the theme from default dark to the system accent colors (which is picked from the windows accent color), the tab and toolbar don't have the same colors as chrome does on Windows 11. Chrome has more darkish and contrasty colors but material fox has more muted greyish colors.

Colors in chrome:
![Screenshot 2024-10-22 224639](https://github.com/user-attachments/assets/4870739d-784a-41f8-9abe-69f1c5f9311e)

Colors in material fox:
![Screenshot 2024-10-22 224700](https://github.com/user-attachments/assets/41c7cb02-cef3-4b56-bb09-2a4097bc8786)

### Steps to reproduce

1. Download the system accent theme CSS file from [here](https://github.com/edelvarden/material-fox-updated/blob/main/examples/theme-system-accent.css) to the `chrome/` folder in Firefox profile folder.
2. Set `userChrome.theme-system-accent` is `about:config` as `true`.
3. Open Firefox.

### Additional info (Optional)

Firefox version: 131.0.3
Operating system: Windows 11

**Comment by edelvarden** [comment]
I can't reproduce your issue. Here's how it looks for me:

![image](https://github.com/user-attachments/assets/f0a2ea21-033b-4b6f-9f30-0442ccb6b5a4)

I simply added the contents of the `theme-system-accent.css` file to the `custom css` and set `userChrome.theme-system-accent` to `true`.

Anyway, this is just an example. I made this color theme for myself and wanted to remove it, but one user insisted I keep it and suggested separating it into different theme file. Feel free to edit the variables as you like.

**Comment by cantunborn** [comment]
Yeah, I *could* edit the variables. This would be a nice feature to add though, making the theme more consistent. Maybe you can check on the default windows 11 wallpaper, the difference is quite large.

Any tips on how you went about matching the colors would also work.

**Comment by edelvarden** [comment]
> Maybe you can check on the default windows 11 wallpaper, the difference is quite large.

![image](https://github.com/user-attachments/assets/2f322927-8b85-4581-9dd6-a28a7553c61c)
![image](https://github.com/user-attachments/assets/35caf8db-faed-421a-a1d8-8ae537632d44)
![image](https://github.com/user-attachments/assets/03a9cd9d-d11b-4aed-a494-1b05fd8f8e33)
![image](https://github.com/user-attachments/assets/2ed6c062-52c9-482e-ad86-71b27e6ed889)


> Any tips on how you went about matching the colors would also work.

I used `color-mix` to mix the default browser colors with the accent color, adjusting the percentages to achieve an acceptable shade. I also toned down the accent color by mixing with white/black, as some colors can be too bright, this is just for compatibility, but you can directly use `AccentColor`, which provides access to the system accent color from CSS.

**Comment by cantunborn** [comment]
I noticed something, if you right click the tab bar on firefox and then enable the title bar in the bottom left, you can see that chrome somehow uses that same color in its tab bar.

Is it possible to access this title bar color in firefox userchrome?

**Comment by cantunborn** [comment]
I just found the color value in this registry location: `HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor`, not sure if it can be accessed inside userChrome.
